frontend/src/screens/channels/Channels.tsx

```diff
@@ -454,53 +443,20 @@ export default function Channels() {
       <div
         className={cn("flex flex-col sm:flex-row flex-wrap gap-3 slashed-zero")}
       >
-        {showHostedBalance && (
-          <Card className="flex flex-col">
-            <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
-              <CardTitle className="text-sm font-medium">
-                Alby Hosted Balance
-              </CardTitle>
-              <Hotel className="h-4 w-4 text-muted-foreground" />
-            </CardHeader>
-            <CardContent className="flex-grow">
-              <div className="text-2xl font-bold">
-                {new Intl.NumberFormat().format(albyBalance.sats)} sats
-              </div>
-              <FormattedFiatAmount amount={albyBalance.sats} />
-            </CardContent>
-            <CardFooter className="flex justify-end space-x-1">
-              <TransferFundsButton
-                variant="outline"
-                channels={channels}
-                albyBalance={albyBalance}
-                onTransferComplete={() =>
-                  Promise.all([
-                    reloadAlbyBalance(),
-                    reloadBalances(),
-                    reloadChannels(),
-                  ])
-                }
-              >
-                Transfer
-              </TransferFundsButton>
-            </CardFooter>
-          </Card>
-        )}
-
         <Card className="flex flex-1 sm:flex-[2] flex-col">
           <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
-            <CardTitle className="font-semibold">Lightning</CardTitle>
```

Comment:
There are a few styling issues here, inconsistency in text sizes and spacing between icons was removed.

Reply:
i rechecked and did few minor style updates. can you recheck again